bindings/rust/wasmedge-sys/src/module.rs

```diff
@@ -0,0 +1,66 @@
+use super::wasmedge;
+use crate::raw_result::{ErrReport, decode_result};
+use std::ffi::CString;
+
+/// Not allowed to be constructed downstream
+pub struct Module {
+    pub(crate) ctx: *mut wasmedge::WasmEdge_ASTModuleContext,
+    _private: (),
+}
+
+impl Default for Module{
+    fn default() -> Self {
+        Self{ ctx: std::ptr::null_mut(), _private: ()}
+    }
+}
+
+struct LoaderContext(*mut wasmedge::WasmEdge_LoaderContext);
+
+
+impl Drop for LoaderContext {
+    fn drop(&mut self) {
+        unsafe { wasmedge::WasmEdge_LoaderDelete(self.0) };
+    }
+}
+
+
+impl Drop for Module {
+    fn drop(&mut self) {
+        unsafe { wasmedge::WasmEdge_ASTModuleDelete(self.ctx) };
+    }
+}
+
+impl Module {
+    fn loader_create(config: &crate::config::Config) -> Result<LoaderContext, ErrReport> {
+        let loader_cxt = unsafe{ 
```

Comment:
please apply rustfmt when you're about to submit

Reply:
Of course, the official pr will do that, I thought you had configured rustfmt in the ci before.